core/experiments/manager.php

```diff
@@ -14,6 +14,8 @@
 	exit; // Exit if accessed directly
 }
 
+require_once __DIR__ . '/experiments-controller.php';
```

Comment:
remove
put the REST code in a controller and register it here

Reply:
@yotamselementor Maybe check what we did with the Filter? https://github.com/elementor/elementor-pro/tree/main/modules/loop-filter/data